controls.py

```diff
@@ -0,0 +1,95 @@
+from testit import *
+import db
+
+
+
+def testTables():
+
+    s = db.get_session()
+
+    # Check Creation
+    A = Account(username = 'JOE',password='ultimatej03',email = 'Joeyjoe@joesph.com',account_token = '555XTZ')
+    S = Student()
+    D = Donor()
+    R = Request(requested_by = 4, amount_needed = 60.00, description = "I NEED A TOAD. TRUST ME.")
+    DM = Donation(to_request= 3, from_donor= 4,amount_given=55.9,charge_token='kEYKey', filled=False)
+
+    s.add(A)
+    s.add(S)
+    s.add(D)
+    s.add(R)
+    s.add(DM) 
+    s.commit()
+
+    print(A, A.__table__, A.id)
+    print(S, S.__table__)
+    print(D, D.__table__)
+    print(R, R.__table__)
+    print(DM, DM.__table__)
+
+    s.close()
+
+
+def createStudent(username, password, email):
+    s = db.get_session()

```

Comment:
Yes, my two base object classes are donors and students, I am considering Student only the accounts capable of submitting requests. I couldn't think of a better label, so name suggestions would be welcome here, I also plan to put doc strings in these functions since they are vvvvvv unclear without them, sorry.